lib/SGN/Controller/AJAX/BrAPI.pm

```diff
@@ -275,6 +275,7 @@ sub _authenticate_user {
 	# Will still throw error if auth is required
 	if ($c->config->{brapi_default_user} && $c->config->{brapi_require_login} == 0) {
 		$user_id = CXGN::People::Person->get_person_by_username($c->dbc->dbh, $c->config->{brapi_default_user});
+		$user_type = 'curator';
```

Comment:
What's the plan for this longer term? Do we eventually want to have something that will work with upstream code?

Reply:
Yeah I imagine we do. It would be nice to create a config file or class that sets the need for authentication on each endpoint, and what role is required, along with the backend mode `brapi_default_user`. I created a card:

https://breedinginsight.atlassian.net/browse/BI-1037